src/styles/blocks/master.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $block-gap;
+  padding: $padding;
+  @include calendar-max-size($calendar-width);
+
+  &__block {
+    @include square($block-size);
+    background-color: $block-color;
+    border: 1px solid #000;
+    border-radius: 5px;
+    transition: transform 1s, color 1s;
+    cursor: pointer;
+  }
+
+  @for $i from 1 through 31 {
+    &__block:nth-child(#{$i})::before {
+      content: "#{$i}";
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      height: 100px;
```

Comment:
Don't use SASS loops for styles that stay the same for all elements of the group.

Reply:
I don't know how I can do it without a loop. I checked  other students' work and they have the same loop